### PR TITLE
Development

### PR DESCRIPTION
**Pull request by avideb1808**

### CODE REVIEW

File: python/tempCodeRunnerFile.py
Comment (akash1047) [inline python/tempCodeRunnerFile.py:1]: why a  this file is commited

File: python/game.py [hunk 1]
```diff
@@ -1,4 +1,14 @@
+import os
+
+# terminal clear command 
+def CLEAR(): # Author Avideb
+    os.system('clear')
```
Comment (akash1047) [inline python/game.py:5]: in windows cmd `clear` command doesn't work

File: python/game.py [hunk 2]
```diff
@@ -1,4 +1,14 @@
+import os
+
+# terminal clear command 
+def CLEAR(): # Author Avideb
```
Comment (akash1047) [inline python/game.py:4]: why this function name is all caps

File: python/game.py [hunk 3]
```diff
@@ -52,10 +74,11 @@ def who_is_the_winner(board, players):
...
 while turn <= 9:
-    if turn & 1:
-        player = p1
-    else:
-        player = p2
+    # if turn & 1:
+    #     player = p1
+    # else:
+    #     player = p2
+    player = p1 if(turn & 1) else p2 # Author -> Avideb
```
Comment (akash1047) [inline python/game.py:81]: keep simple if else block

File: python/tempCodeRunnerFile.py
Comment (akash1047) [inline python/tempCodeRunnerFile.py:1]: why this file is commited

File: python/game.py (hunk 1 above)
Comment (akash1047) [inline python/game.py:5]: `clear` command doesn't work on `windows cmd`

**Review by akash1047 — CHANGES_REQUESTED** [review]

File: python/game.py (hunk 2 above)
Comment (avideb1808) [reply python/game.py:4]: Okay I understand. And reolved it

File: python/game.py (hunk 3 above)
Comment (avideb1808) [reply python/game.py:81]: Done👍🏻

File: python/game.py (hunk 1 above)
Comment (avideb1808) [reply python/game.py:5]: Okay we will add this in future

**Pull request closed without merging**